src/PerfectGym.AutomergeBot/Features/MergingBranches/PullRequestReviewModelHandler.cs

```diff
@@ -0,0 +1,77 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Security.Cryptography;
+using System.Text;
+using System.Xml.Serialization;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+using PerfectGym.AutomergeBot.Features.TempBranchesRemoving;
+using PerfectGym.AutomergeBot.Models;
+using PerfectGym.AutomergeBot.RepositoryConnection;
+
+namespace PerfectGym.AutomergeBot.Features.MergingBranches
+{
+    public interface IPullRequestReviewModelHandler
+    {
+        void Handle(PullRequestReviewInfoModel pullRequestReviewInfoModel);
+    }
+
+    public class PullRequestReviewModelHandler : IPullRequestReviewModelHandler
+    {
+        private readonly IRepositoryConnectionProvider _repositoryConnectionProvider;
+        private readonly ILogger<TempBranchesRemoverPullRequestHandlerPullRequestHandler> _logger;
+        private readonly IOptionsMonitor<AutomergeBotConfiguration> _cfg;
+        private readonly IMergePerformer _mergePerformer;
+
+
+        public PullRequestReviewModelHandler(
+            IRepositoryConnectionProvider repositoryConnectionProvider,
+            ILogger<TempBranchesRemoverPullRequestHandlerPullRequestHandler> logger,
+            IOptionsMonitor<AutomergeBotConfiguration> cfg,
+            IMergePerformer mergePerformer
+            )
+        {
+            _repositoryConnectionProvider = repositoryConnectionProvider;
+            _logger = logger;
+            _cfg = cfg;

```

Comment:
W konstruktorze obiektu wczytać aktualna konfigurację by potem przez całe życie obiektu działać z tę samą konfiguracją. 
Uwaga wynika z tego że takie podejście jest stosowany w dotychczasowym kodzie.
np. tu https://github.com/PerfectGym/AutomergeBot/blob/master/src/PerfectGym.AutomergeBot/Features/TempBranchesRemoving/TempBranchesRemoverPullRequestHandler.cs

 
```suggestion
            _cfg = cfg.CurrentValue;
```